test/spec/modules/parrableIdSystem_spec.js

```diff
@@ -1,15 +1,19 @@
 import { expect } from 'chai';
-import {config} from 'src/config.js';
+import { config } from 'src/config.js';
 import * as utils from 'src/utils.js';
+import { newStorageManager } from 'src/storageManager.js';
+import { getRefererInfo } from 'src/refererDetection.js';
+import { uspDataHandler } from 'src/adapterManager.js';
 import { init, requestBidsHook, setSubmoduleRegistry } from 'modules/userId/index.js';
 import { parrableIdSubmodule } from 'modules/parrableIdSystem.js';
-import { newStorageManager } from 'src/storageManager.js';
+import { server } from 'test/mocks/xhr';
```

Comment:
Can you here extension `.js`?  
I see all other test files that are importing `server` have `.js` extension for `xhr`.